server/worktree/service.go

```
@@ -757,49 +815,90 @@ func (s *Service) DeleteWorktree(ctx context.Context, req serverapi.WorktreeDele
+	if !ok {
+		return nil, serverapi.ErrInvalidControllerLease
+	}
+	if err := collaborative.WithCollaborativeRuntimeEngine(ctx, sessionID, serverapi.SessionRuntimeOperationWorktreeManage, func(*runtimepkg.Engine) error {
+		return errors.New("collaborative runtime guard is unavailable")
+	}); err != nil {
+		return nil, err
+	}
+	return nil, errors.New("collaborative runtime guard is unavailable")
+}
+
+func (s *Service) authorizeMutationRuntime(ctx context.Context, sessionID string, leaseID string) error {
```

golangci-lint (2.12.2)

[error] func (*Service).authorizeMutationRuntime is unused

(unused)